src/aead/aes.rs

```diff
@@ -152,7 +152,7 @@ impl Key {
                 set_encrypt_key!(GFp_vpaes_set_encrypt_key, bytes, key_bits, &mut key)?
             }
 
-            #[cfg(all(not(target_arch = "aarch64"), not(target_env = "sgx")))]
+            #[cfg(all(not(target_arch = "aarch64")))]
```

Comment:
```suggestion
            #[cfg(not(target_arch = "aarch64"))]
```

You can also remove `all()` from every condition as we're only using one condition now.